src/app/loadout/plug-drawer/SelectablePlug.tsx

```diff
@@ -24,12 +24,19 @@ export default function SelectablePlug({
   onPlugRemoved(plug: PluggableInventoryItemDefinition): void;
 }) {
   const defs = useD2Definitions()!;
-  const handleClick = () => {
-    selectable && onPlugSelected(plug);
-  };
+
+  const { handleClick, onClose } = useMemo(() => {
+    const handleClick = () => {
+      selectable && onPlugSelected(plug);
+    };
+
+    const onClose = selected ? () => onPlugRemoved(plug) : undefined;
+
+    return { handleClick, onClose };
+  }, [onPlugRemoved, onPlugSelected, plug, selectable, selected]);
```

Comment:
Would be nicer to just do two separate `useCallback`s here

Reply:
Yeah I was 50/50 on that. Will update